pkg/generators/models/testdata/cases/allof3/api.yaml

```diff
@@ -5,7 +5,11 @@ info:
 
 components:
   schemas:
+    # we expect the fields of Sub and the anonymous
+    # object to be merged into a single struct with
+    # both fields
     Foo:
+      description: this is a foo object
```

Comment:
I added these descriptions to try and reproduce the inconsistent generation, but I have been able to reproduce it in a test yet